apps/aehttp/src/aehttpc_aeternity.erl

```diff
@@ -163,19 +194,45 @@ post_pin_tx(SignedSpendTx, NodeSpec) ->
     %lager:debug("PINNING: wrote to PC tx hash: ~p", [TxHash]),
     encode_child_pin_payload(TxHash).
 
+pin_contract_call(ContractPubkey, PinTx, Who, Amount, Fee, SignModule) ->
+    Nonce = get_local_nonce(Who),
+    {ok, CallData} = aeb_fate_abi:create_calldata("pin", [PinTx]),
+    ABI = 8, % not really nice, what is the supported version of getting the latest ABI version
+    TxSpec =
+        #{  caller_id   => aeser_id:create(account, Who)
+          , nonce       => Nonce
+          , contract_id => aeser_id:create(contract, ContractPubkey)
+          , abi_version => ABI
+          , fee         => Fee
+          , amount      => Amount
+          , gas         => 10
+          , gas_price   => 1000 * min_gas_price()
```

Comment:
gas_price should probably be a parameter if this is an API function. It typically is selected by the caller.

Reply:
With the probable re-design I'm doing, this would be "inside" of the default pinning agent, and not part of an API. would it make sense to make it configurable then?

Reply:
Possibly in the long run... If the chain gets congested, bumping the gas_price is how you get priority. Right now, don't bother I'd say.